src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/shop_list.html.twig

```diff
@@ -0,0 +1,42 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{# The first shop is always displayed entirely and in bold #}
+{% set shops = record[column.options.field] %}
+{% set allShops = shops|join(', ') %}
+<span class="product-shop-list" title="{{ allShops }}">
+  {% set firstShop = shops[0] %}
+  <strong>{{ firstShop }}</strong>
+  {%- if shops|length > 1 -%}
+    ,
```

Comment:
is this comma supposed to be here? just a heads up to make sure its not some typo (looks suspicious :smile:)

Reply:
Yes the list of shops is separated by commas so it's on purpose this one is added if there is more than one shop The other ones via the `join` And to make sure there is no extra space I use the `{%- -%}` twig notation (with extra `-`) so that the spaces are trimmed (I'll add a screenshot to display how it looks like)